src/xunit.performance.logger/app.config

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6"/></startup></configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
```

Comment:
Is it necessary to run against 4.6.1?   Since you're just tricking nuget into treating desktop assemblies as netstandard1.3 I'd assume that with a correct nuspec this wouldn't be needed.